docs/migration_api_keys_to_iam_auth.md

```diff
@@ -0,0 +1,42 @@
+# Migrating from API Key to IAM Auth Access
+
+1. Upgrade to kubectl-hyperkube plugin to 0.6.0 or newer
+
+2. Ensure where you are running kubectl-hyperkube cli commands you are using AWS credentials that you will then whitelist (based upon role, user, or account) in AWS API Gateway Resource Policy
+
```

Comment:
Link to AWS Gateway resource policy docs here?
https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-resource-policies.html

Reply:
Adding link